src/app/landing/students-feedback/students-feedback.component.ts

```diff
@@ -1,25 +1,20 @@
-import { Component, OnInit } from '@angular/core';
-import { StudentsFeedbackService } from '../shared/services/students-feedback.service';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { StudentsFeedbackService } from './students-feedback.service';
 import { StudentFeedback } from '../shared/interfaces/student-feedback';
+import { Observable, Subscription } from 'rxjs';
 
 @Component({
   selector: 'app-students-feedback',
   templateUrl: './students-feedback.component.html',
-  styleUrls: [
-    '../common.css',
-    './students-feedback.component.css'
-  ]
+  styleUrls: ['./students-feedback.component.css']
 })
 export class StudentsFeedbackComponent implements OnInit{
-  studentFeedbacks: StudentFeedback[] = []
+  studentFeedbacks$: Observable<StudentFeedback[]> | undefined = undefined
 
   constructor(private feedbackService: StudentsFeedbackService){}
 
   ngOnInit(): void {
-    this.feedbackService.getFeedbacks()
-      .subscribe(feedbacks => {
-        this.studentFeedbacks = feedbacks
-      })
-      
+    this.studentFeedbacks$ = this.feedbackService.getFeedbacks()
```

Comment:
How? feedbackService is unavailable before constructor execution, can't think of any other way.

Reply:
So, you didn't even try it? So, you didn't even try it? 
This works on the DI mechanism. You can learn more [here](https://angular.io/guide/dependency-injection).
Also, you can try to use a new Angular syntax:
```
private feedbackService: FeedbackService = inject(FeedbackService); // instead of providing property in constructor
```